docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -91,6 +91,25 @@ There are specific pages about contributing to various parts of the Gatsby ecosy
 
 For the Gatsby blog, it's necessary to run your content idea by the Gatsby team before submitting it. For more information, refer to the page on [blog and website contributions](/contributing/blog-and-website-contributions/), including how to propose an idea and setting up the blog to run locally.
 
+## Follow up with reviews and suggestions
+
+After a PR is sent to the Gatsby GitHub repo, the Gatsby core team and the community may suggest modifications to the changes that your PR introduces.
+
+The Gatsby core team reviews and approves every PR that the community sends to make sure that it meets the contribution guidelines of the repo, and to find opportunities for improvement to your PR changes.
```

Comment:
PS: review do not only the core team 

👋

Reply:
I consider learning to be part of core as far as the outside world is concerned, but this could be updated to read:
```suggestion
The Gatsby core and learning teams review and approve every PR that the community sends to make sure that it meets the contribution guidelines of the repo, and to find opportunities for improvement to your PR changes.
```